chcemvediet/apps/inforequests/templates/admin/inforequests/branch/delete_selected_confirmation.html

```diff
@@ -1,27 +0,0 @@
-{% extends "admin/delete_selected_confirmation.html" %}
-{% load amend before after set_attributes from poleno.amend %}
-
-{% block content %}
-  {% amend %}
-    {{ block.super }}
-    {% after path="./ul[last()]" %}
-      {% if outbound %}
-        <p>The following outbound messages will be deleted:</p>
-        <ul>{{ outbound|unordered_list }}</ul>
-      {% endif %}
-      {% if inbound %}
-        <p>The following inbound messages will be marked undecided:</p>
-        <ul>{{ inbound|unordered_list }}</ul>
-      {% endif %}
-    {% endafter %}
-    {% before path=".//form//input[@type='submit']" %}
```

Comment:
@viliambalaz Aby si
```
{% before path=".//form//input[@type='submit']" %}
  {% if delete_warnings %}
    ...
  {% endif %}
  {% if delete_constraints %}
    ...
  {% endif %}
{% endbefore %}
```
nemusel opakovat aj v `inforequests/delete_confirmation.html` aj v `inforequests/delete_selected_confirmation.html` a v buducnosti aj v dalsich suboroch, ktore budu implementovat `BulkDeleteAdminMixin` a nebudu v ramci appky `inforequests`, vysun to do separatneho suboru a v `inforequests/delete_confirmation.html` a `inforequests/delete_selected_confirmation.html` ho iba includni
```
{% block content %}
  {% amend %}
    {{ block.super }}
    ...
    {% include "path/to/foo.html" %}
  {% endamend %}
{% endblock %}
```
Amend by mal fungovat aj ak jeho operacie su includnute z ineho suboru.

Subor nazvi nieco ako `bulk_delete_mixin.html` a daj ho do `poleno/utils/templates/utils/admin/`, kedze je to template tykajuci sa mixinu definovaneho v `poleno/utils/admin.py`. Struktura vo vnutri `poleno/utils/templates` by mala byt taka, ze vsetky templaty budu v adresari `utils/` a v nom rozdelene do poadresarov, z ktoreho suboru v `utils/` adresari pochadzaju.

--
*Poznamka: Komentar je zaveseny k inemu suboru, kedze k `inforequests/delete_confirmation.html` ani `inforequests/delete_selected_confirmation.html` sa nedaju pisat komentare, kedze v nich nedoslo k zmenam.*

Reply:
:ok: